### PR TITLE
[DOMA-2758] Registry schema stich, and IFrameWrapper patch

### PR DESCRIPTION
**Pull request by SavelevMatthew**
Main part is here: https://github.com/open-condo-software/condo-registry-import-billing/pull/2

### CODE REVIEW

**Review by AleX83Xpert — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -823,11 +823,6 @@
   resolved "https://registry.yarnpkg.com/@babel/helper-validator-identifier/-/helper-validator-identifier-7.14.5.tgz#d0f0e277c512e0c938277faa85a3968c9a44c0e8"
   integrity sha512-5lsetuxCLilmVGyiLEfoHBRX8UCFD+1m2x3Rj97WrW3V7H3u4RWRXA4evMjImCsin2J2YT0QaVDGf+z8ondbAg==
 
-"@babel/helper-validator-identifier@^7.14.9":
```
[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply yarn.lock:826]: It changing submodule package.json

[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply yarn.lock:826]: and yarn.lock is common for all submodules

[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply yarn.lock:826]: Resolved: it's left from next downgrade and from other submodules

[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply yarn.lock:826]: Changes approved by @AleX83Xpert

**Pull request merged**